packages/alfa-css/src/syntax/token.ts

```diff
@@ -811,7 +812,10 @@ export namespace Token {
 
   export const { of: whitespace, isWhitespace } = Whitespace;
 
-  export const parseWhitespace = parseToken(isWhitespace);
+  export const parseWhitespace = map(
+    oneOrMore(parseToken(isWhitespace)),
+    ([first,]) => first
```

Comment:
I think the comma is not needed.
```suggestion
    ([first]) => first
```